### PR TITLE
Change to readme for correct installation guidance

### PR DESCRIPTION
**Pull request by ckr123**
The package.json is located in the assets folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ $ git clone https://github.com/hashrocket/tilex
 $ cd tilex
 $ mix deps.get
 $ mix ecto.setup
+$ cd ./assets
```
Comment (jwworth) [inline README.md]: Great catch! Thank you for taking the time to contribute!

Is there a reason to `cd ./assets`, rather than just `cd assets`?

Comment (ckr123) [reply README.md]: Old habits die hard. Changed in 13670c5

**Comment by tmock12** [comment]
Instead of changing directories, running npm install and then changing directories again, we can just run `npm install --prefix assets`

so:
```
$ cd assets
$ npm install
$ cd ..
```

becomes:
```
npm install --prefix assets
```

**Comment by tmock12** [comment]
Actually @ckr123 instead of leaving all three commits, would you mind squashing them into one?

**Comment by jwworth** [comment]
Nice input @tmock12, and thanks for the updates @ckr123. Please squash into one commit and we'll merge this in.

**Comment by ckr123** [comment]
Squashed the commits, glad i could help though it was a really small change. :)
And thank you for a nice product, looking forward to implementing it at work.

**Comment by jwworth** [comment]
Thank you! 👍

**Comment by jwworth** [comment]
Please contact us if you set up an instance and are able to share! Pull requests and issues always welcome.

**Pull request merged**